src/main/java/bhs/devilbotz/RobotContainer.java

```diff
@@ -32,7 +32,7 @@ public class RobotContainer {
 
   private final HashMap<AutonomousModes, Command> autoCommands = new HashMap<>();
 
-  private final DriveTrain driveTrain = new DriveTrain();
+  public final DriveTrain driveTrain = new DriveTrain();
```

Comment:
I see why this was made public.  Is there anyway we can implement this without exposing the driveTrain subsystem?  No biggie if we can't.

Reply:
I'm no OO expert, but I did look at this a bit to see what people do.  `final` means you cannot change it value after initialization. To me a public final variable is same as read only variable.  One way I've seen this done differently is making an accessor method to get the member var needed.  That just seems like lots of extra code that accomplishes same thing.  I like the conciseness using `public final ...`